### ISSUE TITLE
Reference count issue on 2.6.18; can't remove module

### ISSUE THREAD

**Issue by crawfxrd** [datto/dattobd#67]
Fedora Core 6 (2.6.18-1.2798.fc6)
Module fails to unload after creating a snapshot.

Adding the follow to track reference count:
```
LOG_DEBUG("%u: ref count = %d", __LINE__, (int)module_refcount(THIS_MODULE));
```

**Comment by crawfxrd** [comment]
Reference count goes from 1 to 3 after calling `freeze_bdev`
```code
datto: 3033: ref count = 1
datto: freezing block device
datto: 3039: ref count = 3
```

vs Fedora Core 7 (2.6.21-1.3194.fc7)
```
datto: 3033: ref count = 1
datto: freezing block device
datto: 3039: ref count = 1
```

**Comment by crawfxrd** [comment]
We are planning to end support for CentOS 5 in the Datto Agent. This issue will be closed by #131.

**Comment by crawfxrd** [comment]
Won't fix.
